docsteady/spec.py

```diff
@@ -168,6 +168,32 @@ def process_steps(self, test_script):
         return teststeps_sorted
 
 
+def get_lvv_details(key):
+    """
+    get LVV information from Jira
+    :param key: LVV jira Key
+    :return: needed parameter (first stage, only high level req, if available in Jira
+    """
+    lvv = dict()
+    lvv['high_level_req'] = []
+    if key != "":
+        resp = requests.get(
+            Config.ISSUE_URL.format(issue=key),
+            auth=Config.AUTH
+        )
+        if resp.status_code != 200:
+            print("Unable to download")
+            print(resp.text)
+            sys.exit(1)
+        lvv_resp = resp.json()
+        if lvv_resp['fields']['customfield_13515']:
```

Comment:
We should have some kind of documentation or a dictionary somewhere that says what these numbered custom fields are and allows them to be updated in one place if they change for some reason.

Reply:
I have improved the docstring following the link you sent me and added this information.
I am pretty sure that the custom field id will not change in this Jira instance.

However, this makes that docsteady will work only with the current LSST Jira instance, and requires a different approach in case of the Jira instance changes, for example, a new one for operations, or use docsteady in a different project (non LSST).

Reply:
I had put one in config.py, but it's no longer used right now, e.g.
`REQID_FIELD = "customfield_12001"`

I think that makes the config file a dictionary in that case. It also makes it easy to change for a new jira instance, because you could just point to a new config file after finding out the new field number.

Reply:
OK, I adopt the same solution.